porcupine/plugins/find.py

```diff
@@ -141,6 +141,7 @@ def show(self) -> None:
         try:
             selected_text = self._textwidget.get('sel.first', 'sel.last')
         except tkinter.TclError:
+            self.find_entry.selection_range(0, 'end')
             pass   # nothing is selected, leave find entry content as is
```

Comment:
A few things:
- [ ] `pass` can be deleted. It does nothing, and when there was nothing in the `except:` part, it was needed to avoid syntax error.
- [ ] Even though we are deleting the `pass`, I would like to have a comment to mention nothing is selected. It's not obvious by looking at the code.
- [ ] This PR is based on an old version of main branch, because it has single quotes. Latest main has black and double quotes everywhere. Please merge the latest main into it.